x-pack/plugins/security_solution/public/common/components/page/index.tsx

```diff
@@ -31,6 +31,15 @@ export const AppGlobalStyle = createGlobalStyle<{ theme: { eui: { euiColorPrimar
     z-index: 9950 !important;
   }
 
+  /*
+    overrides the default styling of EuiDataGrid expand popover footer to
+    make it a column of actions instead of the default actions row
+  */
+  .euiDataGridRowCell__popover .euiPopoverFooter .euiFlexGroup {
```

Comment:
Does this target every DataGrid popover within the Security plugin? Would it make sense to have a container level class so we're only targeting the Alerts table DataGrid?

Reply:
@mdefazio 
As I pointed yesterday, the popover is appended to the `document.body` when it's opened, so it is impossible to scope it to any container, luckily it brings the `euiDataGridRowCell__popover` class, but this is the more specific scope we can define.
Yes, it will affect all EuiDataGrid _expand_ popovers, but right now we only use EuiDataGrid in the timelines table. Probably we can find a better way to do it when we have a definitive design. 
This was just to match the latest design you shared for 7.15, but we'll need your approval to merge, since it is still blocked by the changes you requested.